src/mergesort.js

```diff
@@ -0,0 +1,33 @@
+export default function mergeSort(array) {
+  if (!Array.isArray(array)) return "We can only sort arrays"
+  
+  if(array.length < 2) return array
+
+  var middle = Math.floor(array.length / 2)
+  var left = array.slice(0, middle)
+  var right = array.slice(middle, array.length)
+
+  return stitch(mergeSort(left), mergeSort(right))
```

Comment:
By convention, this function is usually called `merge`, but I like the name nonetheless.  :P